src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { message } from 'antd';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags.json';
+import { LookService } from '@src/services';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+  id: number;
+};
+
+function StyleTagList({ styleTags, id }: StyleTagListProps) {
+  const [tags, setTags] = useState<StyleTag[]>([...styleTags]);
+
+  const handleCheck = (tag: StyleTag) => {
+    tags.find((t) => t.id === tag.id)
+      ? setTags(tags.filter((t) => tag.id !== t.id))
+      : setTags((prevStayleTags) => [...prevStayleTags, tag]);
+  };
+
+  const handleSave = async () => {
+    await LookService.partialUpdate(id, { styleTags: tags });
+    message.success('성공적으로 저장되었습니다.');
+  };
+
+  return (
+    <>
+      <Wrapper>
+        {styleTagsData.map((tag: StyleTag) => (
+          <Button
+            key={tag.id}
+            title={tag.name}
+            onClick={() => handleCheck(tag)}
+            style={{ margin: '0.4rem' }}
+            type={tags.find((tags) => tags.id === tag.id) ? 'primary' : 'line'}
+            size="small"
+            radius="round"
+          />
```

Comment:
```suggestion
        {STYLE_TAGS.map((styleTag: StyleTag) => (
          <Button
            key={styleTag.id}
            title={styleTag.name}
            onClick={() => handleCheck(styleTag.id)}
            type={selectedTagIds.includes(styleTag.id) ? 'primary' : 'line'}
            size="small"
            radius="round"
          />
```

1. 위 변경점 적용 + 가독성!
2. 배열변수는 복수형으로 네이밍하고, map했을 땐 복수형->단수형으로만 변환합니다!
3. 인라인 스타일을 styled-component로 대체해주세요~

Reply:
66d0864